src/types/onyx/ReportAction.ts

```diff
@@ -10,7 +10,7 @@ import type {Receipt} from './Transaction';
 
 type Message = {
     /** The type of the action item fragment. Used to render a corresponding component */
-    type: string;
+    type?: string;
```

Comment:
Should we be adding a `type` in the test instead of making it optional? Do we know that type might not exist?

Reply:
@kubabutkiewicz are we making it optional only because of the test?

Reply:
Yeah, maybe it will be better to adjust test to include type as `''` instead of making it optional , thanks!